backend/utils/deployShop.js

```diff
@@ -223,62 +237,72 @@ async function deployShop({
       .replace('FAVICON', publicShopConfig.favicon || 'favicon.ico')
   )
 
-  const remotePinners = []
+  // Note: for legacy reasons, the URLs for the IPFS Gateway and API are stored in
```

Comment:
Is it legacy reasons?  Seems like something that doesn't really require encryption.

Reply:
Good point. Hmmm... I don't think we are very clear about what should go as a column vs as encrypted in the config blob :( That's why I said "legacy reasons" :)

It seems we are not consistent: the config right now includes a mix of data that does / does not require encryption. For example things like "dataUrl", "publicUrl", "bigQueryTable" are stored in the config but are not security sensitive...